### PR TITLE
Document all REST API endpoints in gui/README.md

### PR DESCRIPTION
**Pull request by Vavilen**
Docs for:
/wallet
/wallet/transactions
/wallets
/wallets/folderName
/resendUnconfirmedTxns
/blockchain/metadata

### CODE REVIEW

**Comment by Vavilen** [comment]
Fixes #1065

File: src/gui/README.md [hunk 1]
```diff
@@ -171,6 +177,167 @@ result:
...
+	id: Wallet ID
+```
+
+example:
+```bash
+curl http://127.0.0.1:6420/wallet/transactions?id=2017_11_25_e5fb.wlt
+```
+
+result:
+// Returns all pending transanction for all addresses by selected Wallet
```
Comment (gz-c) [inline src/gui/README.md]: Move this comment above the "example:" line

File: src/gui/README.md [hunk 2]
```diff
@@ -171,6 +177,167 @@ result:
...
+    id - Wallet ID [required]
+```
+
+example:
+```bash
+curl http://127.0.0.1:6420/wallet?id=walletId
+```
+
+result:
+```json
```
Comment (gz-c) [inline src/gui/README.md]: format the json with a pretty printer

File: src/gui/README.md [hunk 3]
```diff
@@ -171,6 +177,167 @@ result:
...
+```bash
+curl http://127.0.0.1:6420/wallet/transactions?id=2017_11_25_e5fb.wlt
+```
+
+result:
+// Returns all pending transanction for all addresses by selected Wallet
+
+```json
+[
+   {
```
Comment (gz-c) [inline src/gui/README.md]: format the json with a pretty printer

File: src/gui/README.md [hunk 4]
```diff
@@ -682,9 +849,62 @@ result:
...
+```
+
+example:
+```bash
+curl http://127.0.0.1:6420/resendUnconfirmedTxns
+```
+
+result:
+```json
+{
```
Comment (gz-c) [inline src/gui/README.md]: format the json with a pretty printer

**Comment by gz-c** [comment]
The endpoint for `/wallet/transactions` changed `{"transactions":[...]}`, see https://github.com/skycoin/skycoin/pull/1081
Need to change the doc for this

File: src/gui/README.md (hunk 2 above)
Comment (gz-c) [reply src/gui/README.md]: the format should be the same as the others, with 4 spaces for tabs

**Pull request merged**